docs/conf.py

```diff
@@ -103,6 +104,8 @@
     ('py:class', 'json.encoder.JSONEncoder'),
     ('py:class', 'concurrent.futures._base.Executor'),
     ('py:class', 'concurrent.futures._base.Future'),
+    ('py:class', 'concurrent.futures.process.ProcessPoolExecutor'),
+    ('py:class', 'concurrent.futures.thread.ThreadPoolExecutor'),
```

Comment:
For future reference:
https://github.com/sphinx-doc/sphinx/issues/4826
https://github.com/agronholm/sphinx-autodoc-typehints/issues/38
https://icb-scanpydoc.readthedocs-hosted.com/en/latest/scanpydoc.elegant_typehints.html